snap/snapcraft.yaml

```diff
@@ -0,0 +1,42 @@
+name: smartctl-exporter
```

Comment:
question: why the switch from _ to - ? The payload's name uses the underscore

Reply:
ok

Reply:
```
Bad snapcraft.yaml content:
- Snap names can only use ASCII lowercase letters, numbers, and hyphens, and must have at least one letter (in field 'name')
```

Looks like we can't use underscores here.

Reply:
Didn't know that

Reply:
Me neither until I saw the CI output... TIL :slightly_smiling_face:

Reply:
Reverted to original naming